l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -115,11 +116,15 @@ def saveAttachment(self, fatturapa, number):
         }
         return attach_obj.create(attach_vals)
 
-    def setProgressivoInvio(self, fatturapa):
-
-        file_id = id_generator()
-        while self.env['fatturapa.attachment.out'].file_name_exists(file_id):
```

Comment:
Se si rimuove questo si rischia di generare dei file con nomi già usati

Reply:
Non ricordo se possiamo riutilizzare o meno lo stesso progressivo invio....

Reply:
In generale sicuramente no, per una scartata immagino di sì

Reply:
Ed infatti per questo motivo che riutilizziamo lo stesso progressivo invio, perchè in generale lo facciamo quando c'è un errore :)

Reply:
Se togli questa riga togli il controllo che impedisce a odoo di generare un file già generato. Quindi potresti emettere la fattura 1 con progressivo 12345 e poi la fattura 2 con progressivo sempre 12345

Reply:
Non mi pare ci sia alcun problema, il file viene generato regolarmente. Magari ricontrolliamo meglio

Reply:
Se per la fattura 1 generi il file IT10538570960_12345.xml e poi la fattura 2 generi di nuovo il file IT10538570960_12345.xml, il secondo viene scartato da SDI

Reply:
Ora ho capito dov'era l'errore. Il codice era indentato male e prima andava a controllare l'esistenza dell'attach anche quando ``attach`` era diverso da ``False``. Lo correggo e committo!